index.js

```diff
@@ -5,18 +5,38 @@ import tinycolor from 'tinycolor2';
 import PageData from './components/PageData';
 import Paginator from './components/Paginator';
 
+var {height, width} = Dimensions.get('window');
```

Comment:
Can you change this to `const` and style the curlies as `{ height, width }` please?